File: src/Plexus/Commands/hubCommand.php
<?php 

declare(strict_types=1);

namespace Plexus\commands;

use pocketmine\Player;
use pocketmine\utils\TextFormat as C;
use pocketmine\command\Command;
use pocketmine\command\CommandSender;

use pocketmine\command\defaults\VanillaCommand;

use Plexus\Main;

class hubCommand extends VanillaCommand {
    
  private $plugin;

  public function __construct(Main $plugin){
    $this->plugin = $plugin;
    parent::__construct('hub', 'sends a player back to spawn', '/hub');
    $this->setPermission('plugins.command');
  }

  public function execute(CommandSender $sender, $alias, array $args) : bool {
  if($sender instanceof Player){
    $this->plugin->spawn($sender);
    $sender->addTitle(C::DARK_PURPLE .'Welcome back to Spawn.', C::YELLOW . $sender->getName(), 50, 90, 40);
    return true;
  } else {
    $sender->sendMessage(C::RED .'Command must be run in-game!');
    return false;     
   }
  }
}

File: src/Aether/commands/hubCommand.java
package Aether.commands;

import Aether.AetherPlayer;
import cn.nukkit.Player;
import cn.nukkit.command.CommandSender;
import cn.nukkit.command.defaults.VanillaCommand;
import cn.nukkit.utils.TextFormat;

public class hubCommand extends VanillaCommand {

  private Aether.Main plugin;

  public hubCommand(Aether.Main main, String name){
  super(name);
    this.setPlugin(main);
  }
  
  public Aether.Main getPlugin() {
    return plugin;
  }

  public void setPlugin(Aether.Main plugin) {
    this.plugin = plugin;
  }

  @Override
  public boolean execute(CommandSender player, String alias, String[] args){	
  if(player instanceof Player){
    ((AetherPlayer) player).sendHub();
  } else {
	player.sendMessage(TextFormat.RED +"You can only use /hub in-game!");  
  }
    return true;
  }
}
